homeassistant/components/camera/bloomsky.py

```diff
@@ -0,0 +1,65 @@
+"""
+homeassistant.components.sensor.bloomsky
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Support for BloomSky weather station.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/bloomsky/
+"""
+import logging
+import requests
+import homeassistant.components.bloomsky as bloomsky
+from homeassistant.components.camera import Camera
+
+DEPENDENCIES = ["bloomsky"]
```

Comment:
* specify a version, like `bloomsky==1.0.0`
* add dependency to `requirements_all.txt`